app/controllers/onboarding_controller.rb

```diff
@@ -40,4 +41,8 @@ def validate_params
       obj.require(key)
     end
   end
+
+  def sanitize_real_name
+    params[:real_name].squish!
```

Comment:
Rather than modifying the params with `.squish!, maybe make this method `sanitized_real_name`, use `.squish`, and do `@user.update_attributes(real_name: sanitized_real_name)`